src-electron/main-process/startup.js

```diff
@@ -780,9 +785,6 @@ async function startUpMainInstance(argv, callbacks) {
   } else if (argv._.includes('server')) {
     return startServer(argv, quitFunction)
   } else if (argv._.includes('convert')) {
-    if (argv.zapFiles.length < 1)
-      throw 'You need to specify at least one zap file.'
-    if (argv.output == null) throw 'You need to specify output file.'
```

Comment:
You have deleted the `if ( argv.output == null )` check. There seems to be no replacement for it. Is that ok?

Reply:
To be consistent with the spec, the check should kept, so I will add it back in.

Reply:
done